apps/i18n/common/pt_br.json

```diff
@@ -891,9 +924,12 @@
   "library": "Biblioteca",
   "libraryClassImport": "Importar biblioteca da minha turma",
   "libraryCodeError": "Nós não conseguimos publicar sua biblioteca porque existe um erro no código. Localize o indicador de erros quadrado, em vermelho, e corrija os erros.",
+  "libraryCodeProfanity": "Se aparecer que o seu projeto contém linguagem inapropriada. Por favor, atualize seu projeto e remova a palavra {palavra}.",
```

Comment:
@castro-jorge looks like a translator translated some variables

Reply:
looks like they also aren't making use of the pluralization support we have here; that's not an error, but it's still a bit lower than our desired quality bar